cli/mobycli/scout_suggest.go

```diff
@@ -53,10 +56,14 @@ func displayScoutQuickViewSuggestMsg(image string) {
 	}
 	out := os.Stderr
 	b := color.New(color.Bold)
-	_, _ = fmt.Fprintln(out)
-	_, _ = b.Fprintln(out, "What's Next?")
-	_, _ = fmt.Fprintf(out, "  View summary of image vulnerabilities and recommendations → %s", color.CyanString("docker scout quickview%s", image))
-	_, _ = fmt.Fprintln(out)
+
+	_, _ = b.Fprintln(out, "\nWhat's Next?")
+	if !hubLoggedIn() {
+		_, _ = fmt.Fprintf(out, "  1. Sign in to your Docker account → %s\n", color.CyanString("docker login"))
+		_, _ = fmt.Fprintf(out, "  2. View a summary of image vulnerabilities and recommendations → %s\n", color.CyanString("docker scout quickview%s", image))
+	} else {
+		_, _ = fmt.Fprintf(out, "  View summary of image vulnerabilities and recommendations → %s\n", color.CyanString("docker scout quickview%s", image))
```

Comment:
Nit: "View summary" line 65 or "View a summary"?  line 63

Reply:
Ah, I overlooked the 2nd occurrence when I changed the first case to "a summary"

Reply:
Woops, I copy pasted @dvdksn's suggestion without looking properly 😬